context/app/markdown/docs/assay/seqfish.md

```diff
@@ -1,30 +1,184 @@
 
-**Terms defined in this document**
+## **Definitions:** 
+There are a variety of terms used in this document that may not be familiar to all researchers wanting to make use of the HubMap data. The following figures illustrate several of these terms:
 
-**For Additional Help:** Please contact:
-<!--stackedit_data:
-eyJoaXN0b3J5IjpbMTg2NDc3MzM3Myw3Nzk0MTU1NDgsNDk3OD
-E4ODEwLDczMDk5ODExNl19
--->
\ No newline at end of file
+  
+
+![](https://lh6.googleusercontent.com/H_6Z0jlyG_UpvbDpp9F199714CJphXnOGVArCtImsTLkHa06P2A4NmObkHKb72bPd6dZ_czctwgoe5NeV30SuXnafA6QHHywS59agDAQv-dzW_Zs9K5y9muTaoLpj97MFJuxQyE)
+
+*Figure 4:* Slide Explorer Image is the user defined image on the slide.
+
+The user defined slide will have multiple tiles as numbered in Figure 4. Each of these tiles are considered a single Field of View composed of a 2048 by 2048 pixel image. Tiles are selected by the experiment user manually without overlap. The coordinates of these tiles are used as the global reference to assemble all of the spots from each local tile.
+
+A single Field of View or tile is imaged over many hybridization rounds with three channels for target probes and a final channel of DAPI.
+
+![](https://lh3.googleusercontent.com/L-9j4_4GQucGiyvsyKVNJqxVsB1DreP2w-pTU1F-6B4mF2IbEtfXx80Rof0zjNNhV25hs2fdhDqzc1lbuz1uojfDvWOFcsf4X3k3tpdxHSgqki2DDcvVlCURT1fn-2QxlGIPYDs)![](https://lh5.googleusercontent.com/jW-1zdEs49j9TgCRIoGeI7zFu7qW6-gRviMGh7Njg0_M4AxXJh9qCpZFHP_le85Sh6sAsv03ulexL8PzPXHCz5rAje9TsWSNJv3bwQ2PaMIuyX9iF-iWpXZOzHNB8KS3kgXabq0)
+
+*Figure 5:* Single tile images for one hybridization round, where spots are retrieved in the channels, while the DAPI channel is used for alignment.
+
+## **HuBMAP seqFISH Data States (Levels):**
+The HuBMAP project provides data levels for each seqFISH experiment.
+|**Data State** |  **Description**| **Example File Type** | 
+|--|--|--|
+|  0 | Raw | TIFF
+| 1 |  Processed | TIFF
+| 2 |  Segmentation:  |  TIFF
+| 3 |  Annotation (Cells and Structures): Interpretation  |  TIFF, ZARR, CSV
+
+## **HuBMAP Metadata:** 
+All HuBMAP seqFISH data will have searchable metadata fields. The metadata.tsv can be downloaded from this [Github link](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/seqfish) which also contains field descriptions, and required formats.
+
+## **Associated JSON Files:**
+In addition, all HuBMAP seq data will have an associated json file which may contain the following additional metadata fields:
+
+|**Field** |  **Definition**| **Example**
+|--|--|--|
+|  "version"| Software version used| Micro-Manager-1.4
```

Comment:
@pecan88 : This table is missing row end pipes, but I'll go ahead a merge to get it in.